src/content/blog/astro-290.mdx

```diff
@@ -0,0 +1,141 @@
+---
+title: "Astro 2.9: View Transitions (experimental)"
+description: "Astro 2.9 is out with experimental support for View Transitions, allowing smooth page transitions."
+publishDate: "July 20, 2023"
+authors:
+  - matthew
+coverImage: ""
+socialImage: ""
+lang: "en"
+---
+
+Astro 2.9 is out and features experiment support for [view transitions](https://developer.chrome.com/docs/web-platform/view-transitions/), as well as redirects configuration and improved script bundling.
+
+- **[View Transitions (experimental)](#view-transitions-experimental)**
+- **[Redirects](#redirects)**
+- **[Hoisted script optimization](#hoisted-script-optimization)**
+
+If you already have Astro installed, you can upgrade it to 2.9 by running the `upgrade` command in your project (using your package manager of choice):
+
+```
+npm install astro@latest
+pnpm upgrade astro@latest
+yarn upgrade astro@latest
+```
+
+While you're at it, upgrade any `@astrojs/*` integrations and adapters you have installed, too!
+
+## View Transitions (experimental)
+
+Astro now supports [view transitions](https://developer.chrome.com/docs/web-platform/view-transitions/) through the new `<ViewTransitions />` component and the `transition:animate` (and associated) directives. View transitions are a great fit for content-oriented sites, and we see it as the best path to get the benefits of client-side routing (smoother transitions) without sacrificing the more simple mental model of MPAs.
+
+Enable support for view transitions in Astro 2.9 by adding the experimental flag to your config:
+
+```js
+import { defineConfig } from 'astro/config';
+
+export default defineConfig({
+  experimental: {
+    viewTransitions: true,
+  },
+});
+```
+
+This enables you to use the new APIs added.
+
+### &lt;ViewTransitions /&gt;
+
+This is a component which acts as the _router_ for transitions between pages. Add it to the `<head>` section of each individual page where transitions should occur _in the client_ as you navigate away to another page, instead of causing a full page browser refresh. To enable support throughout your entire app, add the component in some common layout or component that targets the `<head>` of every page.
```

Comment:
```suggestion
This is a component which acts as the _router_ for transitions between pages. Add it to the `<head>` section of each individual page where transitions should occur _in the client_ as you navigate away to another page, instead of causing a full page browser refresh. 

To enable support throughout your entire app (SPA mode), add the component in some common layout or component that targets the `<head>` of every page.
```
Just breaking up a long-ish paragraph, and separating out the line that actually refers to the code sample